docs/UserGuide.md

```diff
@@ -428,6 +494,10 @@ Appointment on 18-11-2024 18:00 with Patient John Doe, S1234567A.
 Appointment on 20-11-2024 18:00 with Patient Hela, S9876543A.
 
 ```
+---
+## Saving/Retrieving Records and AppointmentRecord Data
+<br>
+No Extra command is to be issued to save. All saving and retrieving are automatically performed when needed.
```

Comment:
Perhaps can rephrase this sentence for clarity: Saving and retrieving are performed automatically whenever changes are made, with no additional commands required.

Reply:
ok